src/Microsoft.Restier.EntityFramework/DbDomain.cs

```diff
@@ -20,6 +20,7 @@ namespace Microsoft.Restier.EntityFramework
     /// <summary>
     /// Represents a domain over a DbContext.
     /// </summary>
+    [CLSCompliant(false)]
```

Comment:
Thanks Ray.

From: Ray Yao [mailto:notifications@github.com]
Sent: Thursday, June 25, 2015 12:14 PM
To: OData/RESTier
Cc: Congyong Su
Subject: Re: [RESTier] Integrate EF7 beta6 (#93)


In src/Microsoft.Restier.EntityFramework/DbDomain.cs<https://github.com/OData/RESTier/pull/93#discussion_r33221088>:

> @@ -20,6 +20,7 @@ namespace Microsoft.Restier.EntityFramework

>      /// <summary>

>      /// Represents a domain over a DbContext.

>      /// </summary>

> +    [CLSCompliant(false)]

EF7 DbContext type is not CLS compliant, and that makes a type references it also not compliant. This has been modified to conditionally with EF7.

—
Reply to this email directly or view it on GitHub<https://github.com/OData/RESTier/pull/93/files#r33221088>.